增加标记楼主，叠楼样式调整

atjiu opened the pull request.

tomoya92 committed:
重新发一个pr，那个格式调不回来了 :joy

VitoVan commented on v2excellent.js:
```diff
@@ -222,7 +239,7 @@ function reArrangeComments () {
   });
   $('div[id^="r_"]>table>tbody>tr>td:first-child').attr('width','20');
-  $('body').append('<style>.cell{background-color: inherit;}.cell .cell{border-bottom:none;min-width: 250px;}div[id^="r_"] img.avatar{width:20px;height:20px;border-radius:50%;}div[id^="r_"]>div{margin-left: 5px;}</style>');
+  $('body').append('<style>.cell{background-color: inherit;}.cell .cell{border-bottom:none;min-width: 250px;padding-right:0;}div[id^="r_"] img.avatar{width:20px;height:20px;border-radius:50%;}div[id^="r_"]>div{margin-left: 5px;}</style>');
```
可否帮忙举图例说明为什么要加上 `padding-right:0`

VitoVan commented:
多谢！

1. 可否调整一下 PR 的标题
2. 可否 Rebase，调整一下 Commit Message
3. 可否帮忙举图例说明为什么要加上 `padding-right:0` 
   https://github.com/VitoVan/v2excellent.js/pull/9/files#diff-8ac4bd0c6d4d6526e5332b01b9fca4afR242

atjiu commented:
![qq 20170606174554](https://cloud.githubusercontent.com/assets/6915570/26823500/70dbf234-4ae0-11e7-8f31-da25b8a86319.png)
![qq 20170606174823](https://cloud.githubusercontent.com/assets/6915570/26823506/74da3c10-4ae0-11e7-96f6-b3118b9d89ef.png)

VitoVan commented:
@tomoya92 

🎉 🎉 🎉 

赞，一切妥当，等你修改了 Commit Message 就可以合并啦！

atjiu commented:
修改Commit Message ？意思是我再提交一次吗？为啥对这个还有要求？没问题直接合并不就好了

VitoVan commented:
@tomoya92 嘿，我的好伙计，这里出了些误会，看起来已经使你感到了不悦，很抱歉。

我提到的修改 Commit Message，是指的将下图中的：

![image](https://cloud.githubusercontent.com/assets/1756956/26829355/d841e16a-4af7-11e7-8da2-e2eb9a94f27d.png)


> 重新发一个pr，那个格式调不回来了 :joy

修改为：

> 增加标记楼主，叠楼样式调整

就像你对这个 PR 标题所做的更改一样。

并不是需要你 “再提交一次”。

如果觉得 “修改 Commit Message” 这个描述太过模糊，让人搞不清是要干什么，抱歉了，我指的是类似于下面链接中操作，你可以参考下：

[Git - 重写历史](https://git-scm.com/book/zh/v1/Git-%E5%B7%A5%E5%85%B7-%E9%87%8D%E5%86%99%E5%8E%86%E5%8F%B2)

[Git 的 commit message 写错了，有办法进行修改么？](https://segmentfault.com/q/1010000000761908)

你还有一个问题：

> 为什么对这个有要求？没问题直接合并不就好了

本来想要跟你解释一番，为何要对 Commit Message 有要求，但在看了 [朋也社区 Commits](https://github.com/tomoya92/pybbs/commits/master) 之后便打消了这个念头：

![image](https://cloud.githubusercontent.com/assets/1756956/26829791/64557c1a-4af9-11e7-9321-daf1367f52e4.png)

因为你显然知道一个清晰明了的 Commit Message 所带来的益处。

或许你这个问题是在问：

> 为什么我不可以直接合并，然后自己 Rebase ？

答案是可以，但是我认为需要至少先尊重一下你的意愿：是否真的愿意提交一个PR，带着表意不清楚的 Commit Message？

VitoVan commented:
@tomoya92 忘了说了，我很喜欢你提交的这个功能，很实用！以及右侧的对齐也确实很有必要！

atjiu commented:
老实说，我没找到在哪rebase，如果你不介意的话，可以合并了然后你改一下提交信息就可以了 :joy:

tomoya92 committed:
增加标记楼主，叠楼样式调整，评论图片链接自动转换成图片

增加标记楼主，评论图片链接自动转换成图片   代码来自：https://greasyfork.org/scripts/3452-v2ex%E5%A2%9E%E5%BC%BA%E6%8F%92%E4%BB%B6/code/V2EX%E5%A2%9E%E5%BC%BA%E6%8F%92%E4%BB%B6.user.js

atjiu commented:
又加上了一个功能，评论图片链接自动转换成图片

VitoVan commented:
@tomoya92 https://vitovan.github.io/v2excellent.js/

在页面上加了你的名字~ 有任何疑问随时联系我，谢谢你的 PR！

atjiu commented:
thx
